docs/telemetry/FAQs.md

```diff
@@ -0,0 +1,53 @@
+# FAQs - Frequently asked questions
+
+## Why are you collecting metrics?
+
+We are collecting metrics for all of our projects to assist with prioritizing work and provide proof that our work is valuable. Our prior state had the following issues:
+
+1. Opt-in only metrics for webui and desktop did not send metrics data to our countly server
+    1. We have no insight into the total number of users in our applications
+    2. We had no insight into the percentage of users who declined or accepted metrics
+    3. If a user did not accept or decline metrics, usage was not collected
+2. Lack of singular metrics collection library
+    1. Consent and telemetry patterns were non-existent
+    2. Privacy policy and metrics collection would have needed to be implemented for each library separately.
+    3. Analytics across projects does not allow for normalized comparisons of usage.
+3. Missing telemetry
+    1. We have no insight into the usage of a majority of our projects
+        1. It is understood (not confirmed with data) that two of our top three projects are ipfs-webui & ipfs-desktop, and they had ****some**** telemetry in place.
+    2. We have three separate usecases for ipfs-webui, and are only tracking two of them:
+        1. [webui.ipfs.io](http://webui.ipfs.io) & local - tracked
+        2. webui usage within ipfs-desktop - tracked as desktop usage
+        3. webui bundled with kubo - not tracked
+
+## What data is collected?
+
+In order to standardize the documentation of metrics collection, we will keep a file in each project's repository that
+specifies all metrics data collected by that project. The file will be located at `./docs/telemetry/CollectedData.md`. These documents will provide answers to the following bullet-points:
+
+* This is what will be collected
+* This is what won't be collected
+* Why we are collecting each piece of metric data.
+
+You can read our [Privacy Policy](./PrivacyPolicy.md) and [Collection Policy](./CollectionPolicy.md) for more generic details.
```

Comment:
they won't be next to those other files because they'll be in `./docs/telemetry`.  Also, patterns of bad practice are a reason to stray, not stay, the course.

Note that ipfs-companion also breaks the convention for special files because it is UPPER-KEBAB-CASE, not UPPER_SNAKE_CASE. Also, PRIVACY_POLICY.md does not seem to be a recognized special file. See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/about-community-profiles-for-public-repositories. There is an argument to be made for specifically not following the convention in this case so we do not conflate implications.

Special files are UPPER_SNAKE_CASE, but because these files are specific to the patterns outlined in this PR, I thought camel case was ideal. I'm fine to switch these to follow the standard for github special files if desired.

Reply:
updated the docs to use UPPER_SNAKE_CASE